pkg/limayaml/limayaml.go

```diff
@@ -83,6 +84,13 @@ var (
 	VMTypes    = []VMType{QEMU, VZ, WSL2}
 )
 
+type User struct {
+	Username *string `yaml:"name,omitempty" json:"name,omitempty" jsonschema:"nullable"`
+	Name     *string `yaml:"gecos,omitempty" json:"gecos,omitempty" jsonschema:"nullable"`
```

Comment:
The go name and the yaml name are inconsistent.

I also feel that the short name should be just called "Name" and the gecos field should be called "FullName" or maybe "Comment".

"GECOS" is a remnant from 1960s and no longer meaningful word.

Reply:
I renamed it to `fullName`, but just noticed the `comment` suggestion, and will change to that one instead.

In the CI template we have `Name` for the instance name, and `User` for the user name, so `FullName` does not really fit there (so I left it at `GECOS`), but `Comment` works fine.

Reply:
You didn't like [General Electric Comprehensive Operating Supervisor](https://en.wikipedia.org/wiki/GECOS) ? :-)

Could have gone with `Info` as well, but since nobody looks at it anyway I think `Comment` is fine...

```C
           struct passwd {
               char   *pw_name;       /* username */
               char   *pw_passwd;     /* user password */
               uid_t   pw_uid;        /* user ID */
               gid_t   pw_gid;        /* group ID */
               char   *pw_gecos;      /* user information */
               char   *pw_dir;        /* home directory */
               char   *pw_shell;      /* shell program */
           };
```

Apparently I had some bugs with gecos in "tinycloudinit", and currently it uses `Linux User,,,`

https://github.com/mirror/busybox/blob/1_36_1/loginutils/adduser.c#L199